instructors/4-gis-slides.qmd

```diff
@@ -404,6 +420,23 @@ ggplot(data = old_buildings) +
 ```
 
 ## Basic GIS operations
+As conservationists, we want to create a zone around historical
+buildings where building regulation will have special restrictions to
+preserve historical buildings.
+## Buffer
+Let's say this zone should be 100 meters. In GIS terms, we want to
+create a buffer around polygons. The corresponding function `sf` is
+`st_buffer`, with 2 arguments:
+: the radius of the buffer.  
+
+![Buffer example](https://saylordotorg.github.io/text_essentials-of-geographic-information-systems/section_11/5c7eb6fc4bcbfc5117d778f0aabfded6.jpg)  
+
+## Buffer
+Let's say this zone should be 100 meters. In GIS terms, we want to
+create a buffer around polygons. The corresponding function `sf` is
+`st_buffer`, with 2 arguments:
+-   "x": the polygons around which to create buffers
+-   "dist": the radius of the buffer.
 
 As conservationists, we want to create a zone around historical
 buildings where building regulation will have special restrictions to
```

Comment:
Something seems to be repeated here, like above. Can you please check?

Reply:
Issue to be soleved in https://github.com/carpentries-incubator/r-geospatial-urban/issues/66